packages/cli/src/commands/releasegold/transfer.ts

```diff
@@ -0,0 +1,36 @@
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+import { ReleaseGoldCommand } from './release-gold'
+
+export default class Transfer extends ReleaseGoldCommand {
```

Comment:
Consider calling this command `transfer-dollars` or something like that. Rename the file as well to reflect this